lib/facter/apache_version.rb

```diff
@@ -0,0 +1,8 @@
+Facter.add(:apache_version) do
+  setcode do
+    if Facter::Util::Resolution.which('apachectl')
+      apache_version = Facter::Util::Resolution.exec('apachectl -v 2>&1')
```

Comment:
on older versions of Debuntu, this will not work, because they renamed their apachectl to apache2ctl